.babelrc.js

```diff
@@ -8,9 +8,9 @@ const babelEnv = params => [
 module.exports = api => {
   api.cache(true);
   return {
-    presets: [babelEnv({modules: false}), '@babel/preset-react'],
+    presets: [babelEnv({modules: false}), '@babel/preset-react', '@babel/preset-flow'],
     plugins: [
-      '@babel/plugin-transform-flow-strip-types',
+      '@babel/plugin-transform-runtime',
```

Comment:
This is needed so babel helpers are imported instead of being inlined.

This probably means `@babel/runtime` should be added as peep dependency, am I right?